packages/react-aria-components/src/Popover.tsx

```diff
@@ -173,6 +173,8 @@ function PopoverInner({state, isExiting, UNSTABLE_portalContainer, ...props}: Po
         ref={ref}
         slot={props.slot || undefined}
         style={style}
+        // @ts-ignore - React < 19 compat
+        inert={(isEntering || isExiting) ? 'true' : undefined}
```

Comment:
what is actually the issue?

Reply:
I think this is causing focus not to move into the popover when it opens now. e.g. try opening the column header menu with a keyboard - focus stays on the header instead of going to the menu item.

Reply:
Reverted the inert change and used pointer-events: none instead so it only applies to mouse during the exit animation.